### PR TITLE
first pass at intro to terraform solution/exercise

### PR DESCRIPTION
**Pull request by tkellen**

### CODE REVIEW

File: exercises/introduction-to-terraform/overview.md [hunk 1]
```diff
@@ -2,12 +2,112 @@
...
+terraform apply
+```
+
+> Ensure you have graphviz installed.
+```
+# for osx
+brew install graphviz
+# or debian/ubuntu
+sudo apt-get install graphviz
+# or arch
```
Comment (MattSurabian) [inline exercises/introduction-to-terraform/overview.md:88]: CHALLAH! :fireworks:

Comment (MattSurabian) [inline exercises/introduction-to-terraform/overview.md:88]: really want to go nuts here and add a note about how you oughta be using `yaourt` if you're on Arch, but I'll restrain myself.

**Comment by MattSurabian** [comment]
:+1:

File: exercises/introduction-to-terraform/overview.md (hunk 1 above)
Comment (tkellen) [inline exercises/introduction-to-terraform/overview.md:88]: :dancer:

**Pull request merged**